test/browser.test.js

```diff
@@ -77,40 +74,96 @@ describe('browser support', function() {
       done();
     });
   });
-});
 
-function browserifyTestApp(appDir, strategy, next) {
-  // set default args
-  if (((typeof strategy) === 'function') && !next) {
-    next = strategy;
-    strategy = undefined;
-  }
-  if (!strategy)
-    strategy = 'default';
+  it('supports multiple apps', function(done) {
+    var app1Dir = path.resolve(__dirname, './fixtures/browser-app');
+    var app2Dir = path.resolve(__dirname, './fixtures/browser-app-2');
+
+    var apps = [
+      {
+        appDir: app1Dir,
+        appFile: './app.js',
+        moduleName: 'browser-app'
+      }, {
+        appDir: app2Dir,
+        appFile: './app.js',
+        moduleName: 'browser-app2',
+        appId: 'browserApp2'
+      }
+    ];
 
-  var b = compileStrategies[strategy](appDir);
+    var b = browserify({
+      debug: true,
+    });
+
+    browserifyTestApp(b, apps, function(err, bundlePath) {
+      if (err) return done(err);
+
+      var bundledApps = executeBundledApp(bundlePath, apps);
+      var app1 = bundledApps.defaultApp;
+      var app2 = bundledApps.browserApp2;
 
-  boot.compileToBrowserify(appDir, b);
+      expect(app1.settings).to.have.property('custom-key', 'custom-value');
+      expect(Object.keys(app1.models)).to.include('Customer');
+      expect(app1.models.Customer.settings)
+        .to.have.property('_customized', 'Customer');
+
+      expect(Object.keys(app2.models)).to.include('Email');
```

Comment:
Should this test also verify that `app1` does not contain app2's model and `app2` does not contain `Customer` model?